### DISCUSSION TITLE
Firmware version 5 released - logic analyzer bug fix

### DISCUSSION THREAD

**Discussion by fhdm-dev** [fhdm-dev/scoppy#29] [Announcements]
Version 4 of the firmware has a bug where the pull-downs on on the logic analyzer pins are not disabled. This means the input impedance is way too low and will prevent the analyzer working with high impedance inputs.

A link to the latest firmware can be found [here](https://scoppy.fhdm.xyz/downloads/scoppy-pico-v5.uf2).

**Comment by santlaha** [comment]
🙏👍

**Comment by sjsivo** [comment]
Hello. Nice work but can you share communication protocol and parameters? I will and i need to implement on windows app. Thank you. :-)
